tests/config.py

```diff
@@ -116,12 +120,17 @@ def get_pointer_counter_party_other(cls):
         return Pointer(type_, alias)
 
     @classmethod
-    def get_ip_address(cls):
+    def get_permitted_ips(cls):
         """
-        :rtype: str
+        :rtype: str[]
```

Comment:
Hehe this is not how this is notated in python 😛 

Please rewrite to `list[str]`

Reply:
Thanks! Fixed